File: Add Code Here/C++/tree/Pyramid.c
//C program to print the pyramid pattern
#include <stdio.h>
#include <conio.h>

int main()
{
    int i,j;
    for(i=1; i<=5; i++)
    {
        for(j=1; j<=9; j++)
        {
            if(j>=6-i && j<=4+i)
            {
                printf("*");
            }
            else
            {
                printf(" ");
            }

        }
        printf("\n");
    }
    getch();
}

File: Add Code Here/JAVA/Java-Fundamentals-main/03 - Patterns 1/pyramid.java
public class pyramid {
    public static void main(String[] args) {

        int i,j;
        int a=1;

        for(i=1;i<=5;i++){
            for(j=1;j<=i;j++){
                System.out.print("*");
            }
            System.out.print("\n");
        }

        System.out.println("\n");

        for(i=1;i<=5;i++){
            for(j=1;j<=i;j++){
                System.out.print(a+" ");
                a++;
            }
            System.out.print("\n");
        }
        
    }
}
